common/lib/permissions.js

```diff
@@ -1,4 +1,4 @@
-const policePermissions = [
+const defaultPermissions = [
```

Comment:
I'd probably refrain from creating this as we don't necessarily want to start having the idea of a "default" set of permissions for users. It feels safer to define specifically for each group rather than have the idea of a shared permissions for lots of groups.

Reply:
Refactored so all permissions are explicitly declared and not shared